Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.17
+WORKDIR /go/src/app
+COPY . .
```

Comment:
I think you need to add a `.dockerignore` file in order to ignore `.git`, `.vscode`... otherwise those folders will be unnecessarily copied into docker context

Reply:
Yes, you're right. I'm on it ! :+1: